packages/flutter_tools/lib/src/test/flutter_platform.dart

```diff
@@ -748,15 +750,20 @@ class FlutterPlatform extends PlatformPlugin {
     Uri testUrl,
   }) {
     assert(testUrl.scheme == 'file');
+    final File file = globals.fs.file(testUrl);
     return generateTestBootstrap(
       testUrl: testUrl,
       testConfigFile: findTestConfigFile(globals.fs.file(testUrl)),
       host: host,
       updateGoldens: updateGoldens,
-      nullSafety: extraFrontEndOptions?.contains('--enable-experiment=non-nullable') ?? false,
+      languageVersionHeader: determineLanguageVersion(
+        file,
+        _packageConfig[flutterProject?.manifest?.appName],
```

Comment:
Could `_packageConfig` be `null` if `loadPackageConfigWithLogging` fails?

Reply:
No, it will throw a tool exit if `throwOnError` is true (which it is by default)